### PR TITLE
stats: fix converting duration to timestamp

### LINKED ISSUE #8115: statistics update error for deleted tables
## Bug Report

1. What did you do?
some tables dropped

2. What did you expect to see?
statistics shouldn't be updated for deleted tables

3. What did you see instead?
tidb-server.log is flooded with these logs:

```

2018/10/31 11:59:24.241 adapter.go:370: [debug] [QUERY] cost_time:8.946046ms process_time:2ms wait_time:4ms request_count:2 total_keys:3 processed_keys:2 succ:true con:3115915 user:web@192.168.1.101 txn_start_ts:403952989420650512 database:ndb sql:UPDATE w_msg SET comment_num=?,read_num=?,like_num=? WHERE hash_id=?
2018/10/31 11:59:24.247 txn.go:190: [debug] [kv]3115915 txnLatches disabled, 2pc directly:<nil>
2018/10/31 11:59:24.251 server.go:316: [info] con:3115915 close connection
2018/10/31 11:59:24.274 table.go:225: [debug] We cannot find column id 5 in table info rt_dy_article now. It may be deleted.
2018/10/31 11:59:24.274 table.go:225: [debug] We cannot find column id 7 in table info rt_dy_article now. It may be deleted.
2018/10/31 11:59:24.282 table.go:225: [debug] We cannot find column id 7 in table info rt_dy_userinfo_extract now. It may be deleted.
2018/10/31 11:59:24.282 table.go:225: [debug] We cannot find column id 22 in table info rt_dy_userinfo_extract now. It may be deleted.
2018/10/31 11:59:24.480 table.go:225: [debug] We cannot find column id 31 in table info rt_article_detail_app_old now. It may be deleted.
2018/10/31 11:59:24.485 table.go:225: [debug] We cannot find column id 4 in table info rt_x_article_id_old now. It may be deleted.
2018/10/31 11:59:24.524 table.go:225: [debug] We cannot find column id 5 in table info wv_dy_user_daily_sum_data now. It may be deleted.
2018/10/31 11:59:24.524 table.go:225: [debug] We cannot find column id 6 in table info wv_dy_user_daily_sum_data now. It may be deleted.
2018/10/31 11:59:24.524 table.go:225: [debug] We cannot find column id 7 in table info wv_dy_user_daily_sum_data now. It may be deleted.
2018/10/31 11:59:24.524 table.go:225: [debug] We cannot find column id 8 in table info wv_dy_user_daily_sum_data now. It may be deleted.
2018/10/31 11:59:24.524 table.go:225: [debug] We cannot find column id 9 in table info wv_dy_user_daily_sum_data now. It may be deleted.
2018/10/31 11:59:24.524 table.go:225: [debug] We cannot find column id 10 in table info wv_dy_user_daily_sum_data now. It may be deleted.
2018/10/31 11:59:24.682 table.go:225: [debug] We cannot find column id 12 in table info t_spi_task now. It may be deleted.
2018/10/31 11:59:24.708 ddl_worker.go:133: [debug] [ddl-worker 2, tp add index] wait 1s to check DDL status again
2018/10/31 11:59:24.708 ddl_worker.go:133: [debug] [ddl-worker 1, tp general] wait 1s to check DDL status again
2018/10/31 11:59:24.709 ddl.go:262: [debug] [ddl] it's the DDL owner true, self ID 0af7f99f-ab8f-4387-9679-c866487a705d
2018/10/31 11:59:24.709 ddl.go:262: [debug] [ddl] it's the DDL owner true, self ID 0af7f99f-ab8f-4387-9679-c866487a705d
2018/10/31 11:59:24.797 table.go:225: [debug] We cannot find column id 25 in table info test1 now. It may be deleted.
2018/10/31 11:59:24.797 table.go:225: [debug] We cannot find column id 31 in table info test1 now. It may be deleted.
2018/10/31 11:59:24.797 table.go:140: [debug] We cannot find index id 3 in table info test1 now. It may be deleted.
2018/10/31 11:59:25.274 table.go:225: [debug] We cannot find column id 7 in table info t_zh_article_extract now. It may be deleted.
2018/10/31 11:59:25.274 table.go:225: [debug] We cannot find column id 8 in table info t_zh_article_extract now. It may be deleted.
2018/10/31 11:59:25.274 table.go:225: [debug] We cannot find column id 9 in table info t_zh_article_extract now. It may be deleted.
2018/10/31 11:59:25.285 table.go:225: [debug] We cannot find column id 10 in table info wv_wxuser_extinfo1 now. It may be deleted.
github.com/pingcap/tidb/vendor/github.com/pkg/errors.AddStack
	/go/src/github.com/pingcap/tidb/vendor/github.com/pkg/errors/errors.go:173
github.com/pingcap/tidb/terror.(*Error).GenWithStack
	/go/src/github.com/pingcap/tidb/terror/terror.go:224
github.com/pingcap/tidb/types.overflow
	/go/src/github.com/pingcap/tidb/types/etc.go:213
github.com/pingcap/tidb/types.ConvertUintToUint
	/go/src/github.com/pingcap/tidb/types/convert.go:120
github.com/pingcap/tidb/types.(*Datum).convertToUint
	/go/src/github.com/pingcap/tidb/types/datum.go:878
github.com/pingcap/tidb/types.(*Datum).ConvertTo
	/go/src/github.com/pingcap/tidb/types/datum.go:679
github.com/pingcap/tidb/statistics.(*Handle).histogramFromStorage
	/go/src/github.com/pingcap/tidb/statistics/histogram.go:309
github.com/pingcap/tidb/statistics.(*Handle).columnStatsFromStorage
	/go/src/github.com/pingcap/tidb/statistics/table.go:195
github.com/pingcap/tidb/statistics.(*Handle).tableStatsFromStorage
	/go/src/github.com/pingcap/tidb/statistics/table.go:265
github.com/pingcap/tidb/statistics.(*Handle).Update
	/go/src/github.com/pingcap/tidb/statistics/handle.go:149
github.com/pingcap/tidb/domain.(*Domain).updateStatsWorker
	/go/src/github.com/pingcap/tidb/domain/domain.go:756
runtime.goexit
	/usr/local/go/src/runtime/asm_amd64.s:2361.
```

4. What version of TiDB are you using (`tidb-server -V` or run `select tidb_version();` on TiDB)?

```
Release Version: v2.1.0-rc.3-17-g3e3b905
Git Commit Hash: 3e3b90580f588eb0b3c72b11f635e6ec9b2384db
Git Branch: release-2.1
UTC Build Time: 2018-10-18 05:02:20
GoVersion: go version go1.10.4 linux/amd64
Race Enabled: false
TiKV Min Version: 2.1.0-alpha.1-ff3dd160846b7d1aed9079c389fc188f7f5ea13e
Check Table Before Drop: false
```

### PR DESCRIPTION
**Pull request by lamxTyler**
### What problem does this PR solve? 

The deleted stats is not removed for a long time.
Fix #8115 
### What is changed and how it works?

We will delete the stats that have been deleted in 10 stats-lease ago. But when converting the stats lease to timestamp, which is the version info used in stats, we forgot to divide the million second unit, thus resulting in long stale time.

### Check List 

Tests 

 - Unit test

Code changes

 - Has exported function/method change

Side effects

 - None

Related changes

 - Need to cherry-pick to the release branch

PTAL @zz-jason @winoros @lzmhhh123

### CODE REVIEW

**Comment by shenli** [comment]
Will this affect the release-2.1?

**Comment by lamxTyler** [comment]
@shenli Yes, I will cherry-pick it when this is merged.

File: statistics/handle.go [hunk 1]
```diff
@@ -109,6 +109,11 @@ func (h *Handle) GetQueryFeedback() []*QueryFeedback {
 	return h.feedback
 }
 
+// DurationToTS converts duration to timestamp.
+func DurationToTS(d time.Duration) uint64 {
+	return oracle.ComposeTS(int64(d)/int64(time.Millisecond), 0)
```
Comment (zz-jason) [inline statistics/handle.go]: how about: 
```go
d.Nanoseconds()/int64(time.Millisecond)
```

**Pull request merged**